pkgs/tools/backup/mtx/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "mtx-1.3.12";
+
+  src = fetchurl {
+    url = "mirror://gentoo/distfiles/${name}.tar.gz";
+    sha256 = "0261c5e90b98b6138cd23dadecbc7bc6e2830235145ed2740290e1f35672d843";
+  };
+
+  doCheck = false;
+
+  meta = {
+    description = "Command line utility to manipulate tape robots";
+    longDescription = ''
+The mtx command controls single or multi-drive SCSI media changers such as tape changers, autoloaders, tape libraries, or optical media jukeboxes. It can also be used with media changers that use the 'ATTACHED' API, presuming that they properly report the MChanger bit as required by the SCSI T-10 SMC specification.
```

Comment:
I'm assuming you mean thet longDescription.  If I'm missing something else please let me know.

First commit to this repo so may be missing things that are obvious to you but not to me.